index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+
+<body>
+    <header class="header">
+        <div class="logo-container">
+            <a href="#" class="my-logo">My Logo</a>
+        </div>
+        <nav class="menu">
+            <a href="#"> <img src="./images/Union.png" alt="menu-icon" class="hamberger-menu"></a>
```

Comment:
- [OPTIONAL] :thinking: I think it would be better to reduce the padding in the header to match the Figma design.
![header](https://user-images.githubusercontent.com/97834160/212031129-335ad26c-82c4-4cf8-868a-a9dbfa651ebc.png)